assets/css/main.css

```
@@ -4,6 +4,51 @@
+        30%, 50%, 70% {
+            transform: translateX(-6px);
+        }
+
+        40%, 60% {
+            transform: translateX(6px);
+        }
+    }
+}
+
+.text-border-white {
+    color: theme('colors.black.DEFAULT');
```

Biome (1.9.4)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)